x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/index.tsx

```diff
@@ -18,6 +19,8 @@ export const EndpointPackageCustomExtension = memo<PackageCustomExtensionCompone
         <FleetTrustedAppsCardWrapper {...props} />
         <EuiSpacer />
         <FleetEventFiltersCard {...props} />
+        <EuiSpacer />
```

Comment:
Can you follow up with @caitlinbetz or @bfishel on what the order of these should be?
I'm thinking alphabetical by Artifact type, but they might have something else in mind.

Reply:
I consulted with @bfishel and she agrees with the current order that follows the navigation bar.